### PR TITLE
fix/ntgl-414 footer fixes

### PR DESCRIPTION
**Pull request by jeremyle56**
- Fixed light mode sponsor logos
- Fixed line wrapping for smaller screens in footer
- Adjusted spacing surrounding sponsor block
- Horizontally aligned footer text and block

image
image

### CODE REVIEW

File: client/src/components/Sponsors.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Box, Link, Stack } from '@mui/material';
+import styled from '@mui/system/styled';
```
Comment (Rayahhhmed) [inline client/src/components/Sponsors.tsx:2]: Can you fix the imports to sort alphabetically

File: client/src/components/footer/FooterInfo.tsx [hunk 2]
```diff
@@ -13,18 +13,42 @@ interface TitleTextProps {
...
 
 const TitleText = styled('h2')<TitleTextProps>`
   font-size: ${({ isLarge }) => (isLarge ? '18px' : '15px')};
+
+  @media (max-width: 450px) {
+    font-size: ${({ isLarge }) => (isLarge ? '16px' : '14px')};
+  }
+`;
+
+const NotanglesContainer = styled('div')`
```
Comment (Rayahhhmed) [inline client/src/components/footer/FooterInfo.tsx]: nit `NotanglesContainer` can mean many things, be specific ie `NotanglesFooterInfoContainer` or something

File: client/src/components/footer/FooterLinks.tsx [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
-import MailOutlineOutlinedIcon from '@mui/icons-material/MailOutlineOutlined';
 import { Link, Stack } from '@mui/material';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
 import React from 'react';
 
+const DiscordIcon = (props: SvgIconProps) => {
+  return (
+    <SvgIcon {...props}>
+      <path d="M20.32 4.37a19.8 19.8 0 0 0-4.89-1.52.07.07 0 0 0-.08.04c-.2.38-.44.87-.6 1.25a18.27 18.27 0 0 0-5.5 0c-.16-.4-.4-.87-.6-1.25a.08.08 0 0 0-.09-.04 19.74 19.74 0 0 0-4.88 1.52.07.07 0 0 0-.04.03A20.26 20.26 0 0 0 .1 18.06a.08.08 0 0 0 .03.05 19.9 19.9 0 0 0 6 3.03.08.08 0 0 0 .08-.02c.46-.63.87-1.3 1.22-2a.08.08 0 0 0-.04-.1 13.1 13.1 0 0 1-1.87-.9.08.08 0 0 1 0-.12l.36-.3a.07.07 0 0 1 .08 0 14.2 14.2 0 0 0 12.06 0 .07.07 0 0 1 .08 0l.37.3a.08.08 0 0 1 0 .12 12.3 12.3 0 0 1-1.88.9.08.08 0 0 0-.04.1c.36.7.78 1.36 1.23 2a.08.08 0 0 0 .08.02c1.96-.6 3.95-1.52 6-3.03a.08.08 0 0 0 .04-.05c.5-5.18-.84-9.68-3.55-13.66a.06.06 0 0 0-.03-.03zM8.02 15.33c-1.18 0-2.16-1.08-2.16-2.42 0-1.33.96-2.42 2.16-2.42 1.21 0 2.18 1.1 2.16 2.42 0 1.34-.96 2.42-2.16 2.42zm7.97 0c-1.18 0-2.15-1.08-2.15-2.42 0-1.33.95-2.42 2.15-2.42 1.22 0 2.18 1.1 2.16 2.42 0 1.34-.94 2.42-2.16 2.42Z" />
```
Comment (Rayahhhmed) [inline client/src/components/footer/FooterLinks.tsx]: wot is this path 💀

Comment (jeremyle56) [reply client/src/components/footer/FooterLinks.tsx]: Material Icons doesn't have a discord icon. Would you prefer to move this out to a file?

File: client/src/components/Sponsors.tsx (hunk 1 above)
Comment (jeremyle56) [reply client/src/components/Sponsors.tsx:2]: Sorry what do you mean? Is that not alphabetical?

File: client/src/components/footer/FooterInfo.tsx (hunk 2 above)
Comment (jeremyle56) [reply client/src/components/footer/FooterInfo.tsx]: @Shzmj 
clown-face_1f921

File: client/src/components/footer/FooterLinks.tsx (hunk 3 above)
Comment (Rayahhhmed) [reply client/src/components/footer/FooterLinks.tsx]: yes pls, move to a file :)

**Review by Rayahhhmed — APPROVED** [review]

**Review by Shzmj — APPROVED** [review]
LGTM

**Pull request merged**